### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@vikivivi please review

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -27071,6 +27123,7 @@ responc->response, respond,
 responce->response
 responces->responses
 responde->respond, response, responds, responded, responder,
+respones->response, responses,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:27126]:
```text
respone->response
```
* https://github.com/search?q=respone&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -31707,6 +31763,7 @@ tocksen->toxin
 todya->today
 toekn->token
 togehter->together
+togeter->together
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:31766]:
```text
togeterness->togetherness
```
* https://github.com/search?q=togeterness&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi Added in [2b87890](https://github.com/codespell-project/codespell/pull/2385/commits/2b878906b46c399d39c9f42fcb44dd707e75e120). Please review

**Pull request merged**